docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -1,161 +1,262 @@
 # Pagination
 
-Pagination is a common component supporting multiple pages. It shows the number of pages available and the currently active page.
+
+Pagination is a component for making navigation through multipage windows more user-friendly.
+It can be used to display the number of available pages and marking the currently viewed one.
+The figure below illustrates the use of a simple pagination - it shows 5 available pages, of which the first one is currently displayed.
 
 ![Pagination](./media/Pagination.png)
 
-## Create with property
+Pagination can be customized for size and the background color as well as for using images or solid colors for indicators.
+Navigation within the pages occurs by attaching to it the window key event as described at the end of this section.
 
-To create a Pagination using property, follow these steps:
 
-1. Create Pagination using the default constructor:
+In the following sections on this guide one can find:
+- [the main pagination properties](#paginationProperties)
+- how to create a pagination
+   - [using properties](#createWithProperty)
+   - [using style](#createWithStyle)
+   - [using a custom style](#createWithCustomStyle)
+- [how to connect pagination with a KeyEvent](#connectWithKeyEvent)
 
-    ```cs
-    pagination = new Pagination();
-    ```
 
-2. Set the Pagination property:
+<a name="paginationProperties"></a>
+## Pagination properties
+
+The following table lists the properties specific to the Pagination class:
+
+**Table: Pagination properties**
+
+| Property                  | Type               | Description                                   |
+|---------------------------|--------------------|-----------------------------------------------|
+| `IndicatorSize`           | `Size`             | The absolute size of the indicator.           |
+| `IndicatorSpacing`        | `int`              | The space between the indicators (in pixels). |              
```

Comment:
```suggestion 
| `IndicatorSpacing`        | `int`              | The space between the indicators, it is measured in pixels. |

Reply:
can I write shorter? :
The space between the indicators measured in pixels.

Reply:
Yes, you can. It is fine.